### DISCUSSION TITLE
Welcome to Journ-1-Social-studies Discussions!

### DISCUSSION THREAD

**Discussion by Super45coder** [Journ12-Journals/Journ-1-Social-studies#3] [Announcements]
👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by N3RDIUM** [comment]
Hi There!
